fail2ban/helpers.py

```diff
@@ -137,3 +142,48 @@ def splitcommaspace(s):
 	if not s:
 		return []
 	return filter(bool, re.split('[ ,]', s))
+
+
+class BgService(object):
+	"""Background servicing
+
+	Prevents memory leak on some platforms/python versions, 
+	using forced GC in periodical intervals.
```

Comment:
I think I left this comment before:  please add urls right here pointing to those issues in Python, so there is a context for requiring this